src/main/java/training_day5/HW/School.java

```diff
@@ -0,0 +1,124 @@
+package training_day5.HW;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class School {
+
+    private String name;
+    private List<Teacher> teachers = new ArrayList<>();
+    private List<Student> students = new ArrayList<>();
+
+    public School(String name){
+        this.name = name;
+    }
+
+    public void setName(String name){
+        this.name = name;
+    }
+
+    public String getName(){
+        return this.name;
+    }
+
+    protected void registerStudent(Student a) {
+        this.students.add(a);
+    }
+
+    protected void registerTeacher(Teacher a) {
+        this.teachers.add(a);
+    }
+
+    public void printTeachersAndStudents(){
+        System.out.println("*****Teachers*****");
```

Comment:
Read about toString() and what it is used for. You can override it in your classes and use it here.